apps/editors/decorators.py

```diff
@@ -33,6 +33,23 @@ def wrapper(request, *args, **kw):
     return wrapper
 
 
+def unlisted_addons_reviewer_required(f):
+    """Requires the user to be logged in as an unlisted addons reviewer or
+    admin.
+
+    An unlisted addons reviewer is someone who is in a group with the following
+    permission: Addons:ReviewUnlisted.
+
```

Comment:
There shouldn't be a newline after opening quotes for a docstring (single or multi-line), according to the [pep](https://www.python.org/dev/peps/pep-0257/#id17). However, the first line should also fit on a single line... which isn't the case here.

Also the wording isn't correct: taken from the pep
```The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...". ```

I'll change those.